assets/keymaps/default-linux.json

```diff
@@ -559,7 +559,6 @@
       "shift-insert": "project_panel::Paste",
       "ctrl-alt-c": "project_panel::CopyPath",
       "alt-ctrl-shift-c": "project_panel::CopyRelativePath",
-      "f2": "project_panel::Rename",
```

Comment:
<img width="616" alt="image" src="https://github.com/zed-industries/zed/assets/2690773/53d1aa49-31e5-4075-ae1c-d9e877c156e5">

Any reason to remove in for Linux?

Reply:
No, that was an oversight on my part. I do find it a bit odd that we have both Enter and F2 for Linux but I'm more than happy to leave it as is.